x-pack/legacy/plugins/apm/public/components/shared/TransactionBreakdown/TransactionBreakdownGraph/index.tsx

```diff
@@ -4,10 +4,62 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React from 'react';
+import React, { useMemo, useCallback } from 'react';
+import numeral from '@elastic/numeral';
+import { NOT_AVAILABLE_LABEL } from '../../../../../common/i18n';
+import { Coordinate } from '../../../../../typings/timeseries';
+import { TransactionLineChart } from '../../charts/TransactionCharts/TransactionLineChart';
+import { asPercent } from '../../../../utils/formatters';
+import { unit } from '../../../../style/variables';
 
-const TransactionBreakdownGraph: React.FC<{}> = () => {
-  return <div />;
+interface Props {
+  timeseries: Array<{
+    name: string;
+    color: string;
+    values: Array<{ x: number; y: number }>;
+  }>;
+}
+
+const TransactionBreakdownGraph: React.FC<Props> = props => {
+  const { timeseries } = props;
+
+  const series: React.ComponentProps<
+    typeof TransactionLineChart
+  >['series'] = useMemo(
+    () => {
+      return timeseries.map(timeseriesConfig => {
+        return {
+          title: timeseriesConfig.name,
+          color: timeseriesConfig.color,
+          data: timeseriesConfig.values,
+          type: 'area',
+          hideLegend: true
+        };
+      }, {});
```

Comment:
the last arg `{}` isn't needed since there's no reference to `this` in the map callback function

Reply:
aye, think this was a reduce before and I forgot to remove the second argument. thanks!